test/integration/base.py

```diff
@@ -333,6 +333,14 @@ def assertTableDoesNotExist(self, table):
             0
         )
 
+    def assertTableDoesExist(self, table):
+        columns = self.get_table_columns(table)
+
+        self.assertGreater(
+            len(columns),
+            0
+        )
```

Comment:
this is cool. not directly related to this branch but i don't think this works on snowflake. it'd be great to test this across warehouses.